packages/trackerless-network/src/logic/StreamrNode.ts

```diff
@@ -217,7 +217,7 @@ export class StreamrNode extends EventEmitter<Events> {
             forwardingNode
         )
         entryPoints = knownEntryPoints.concat(discoveryResult.discoveredEntryPoints)
-        await layer1.joinDht(sampleSize(entryPoints, NETWORK_SPLIT_AVOIDANCE_LIMIT), false, knownEntryPoints.length > 0)
+        await layer1.joinDht(sampleSize(entryPoints, NETWORK_SPLIT_AVOIDANCE_LIMIT), true, knownEntryPoints.length > 0)
```

Comment:
Could write a test for this as none of our test caught this bug

Reply:
The random joins are not strictly necessary when joining streams as we care more about knowledge of nearby peers but it does improve the reliability of the stream if nodes know of more candidates to connect to

Reply:
https://linear.app/streamr/issue/NET-1078/test-case-for-random-join